lessons/beginners/list/index.md

```diff
@@ -381,112 +425,35 @@ print(slova)
 ```
 
 Metoda `split` umí brát i argument.
-Pokud ho předáme, místo mezer (a nových řádků)
-se řetězec „rozseká” daným oddělovačem.
-Takže když máme nějaká data oddělená čárkami,
-není nic jednoduššího než použít `split` s čárkou:
+Pokud ho předáš řetězec „rozseká” daným oddělovačem
+(místo mezer a nových řádků).
+Takže když máš nějaká data oddělená čárkami,
+použíj `split` s čárkou:
 
 ```python
 zaznamy = '3A,8B,2E,9D'.split(',')
 print(zaznamy)
 ```
 
-Chceme-li spojit seznam řetězců zase dohromady
-do jediného řetězce, použijeme metodu
-`join` (angl. *spojit*).
+Chceš-li spojit seznam řetězců zase dohromady
+do jediného řetězce, použij metodu `join` (angl. *spojit*).
 Pozor, tahle metoda se volá na *oddělovači*,
-tedy řetězci, kterým se jednotlivé kousky „slepí”
-dohromady; a jako argument bere seznam jednotlivých
-řetězců.
+tedy na řetězci, kterým se jednotlivé kousky „slepí” dohromady.
+Seznam jednotlivých řetězců bere jako argument.
```

Comment:
Nějakou poznámku pod čarou, proč tomu tak je? Toto notoricky mate úplně každého.

Reply:
Když se to pokusím vysvětlit na téhle úrovni, tak se do toho moc zamotám :(